lib/oelite/profiling.py

```diff
@@ -98,9 +97,10 @@ def print_deferred(self):
         for p in self.deferred:
             p.print_delta()
 
-    def __init__(self, name):
+    def __init__(self, name, print_walltime=True):
```

Comment:
I suppose I could do the teaching in a separate commit, but while the commit message doesn't mention profiling.py by that name, it does refer to oelite.profiling.